develop-docs/sdk/telemetry/traces/dynamic-sampling-context.mdx

```diff
@@ -60,6 +61,8 @@ The rest of the context attributes, `trace_id`, `public_key`, and `sample_rate`,
 | `release`                   | string | The release name as specified in client options.                                                                             | `myapp@1.2.3`, `1.2.3`, `2025.4.107` | required                             |
 | `environment`               | string | The environment name as specified in client options.                                                                         | `production`, `staging`              | required                             |
 | `transaction`               | string | The transaction name set on the scope. **Only include** if name has [good quality](#note-on-good-quality-transaction-names). | `/login`, `myApp.myController.login` | required (if known and good quality) |
+| `org`                       | string | The org ID parsed from the DSN or received by a downstream SDK.                                                              | `1`                                  | required                             |
+| `sample_rand`               | string | A trully random number originating from the head of trace SDK.                                                               | `0.5`                                | required                             |
```

Comment:
Does it have to be a true random number? (I guess not all systems can create a true random number)

Reply:
Let's just use whatever random number we can get.